interface/patient_file/pos_checkout.php

```diff
@@ -166,969 +160,966 @@ function generate_receipt($patient_id, $encounter = 0)
-    <?php
-    $query1112 = "SELECT * FROM list_options where list_id=?  ORDER BY seq, title ";
-    $bres1112 = sqlStatement($query1112, array('payment_method'));
-    while ($brow1112 = sqlFetchArray($bres1112)) {
-        if ($brow1112['option_id']=='electronic' || $brow1112['option_id']=='bank_draft') {
-            continue;
-        }
-
-        echo "<option value='".attr($brow1112['option_id'])."'>".text(xl_list_label($brow1112['title']))."</option>";
-    }
+    // Now write a data entry form:
+    // List unbilled billing items (cpt, hcpcs, copays) for the patient.
+    // List unbilled product sales for the patient.
+    // Present an editable dollar amount for each line item, a total
+    // which is also the default value of the input payment amount,
+    // and OK and Cancel buttons.
     ?>
-   </select>
-  </td>
- </tr>
-
- <tr>
-  <td>
-    <?php echo xlt('Check/Reference Number'); ?>:
-  </td>
-  <td>
-   <input type='text' name='form_source' size='10' value=''>
-  </td>
- </tr>
-
- <tr>
-  <td>
-    <?php echo xlt('Amount Paid'); ?>:
-  </td>
-  <td>
-   <input type='text' name='form_amount' size='10' value='0.00'>
-  </td>
- </tr>
-
- <tr>
-  <td>
-    <?php echo xlt('Posting Date'); ?>:
-  </td>
-  <td>
-   <input type='text' size='10' class='datepicker' name='form_date' id='form_date'
-    value='<?php echo attr($inv_date) ?>'
-    title='yyyy-mm-dd date of service' />
-  </td>
- </tr>
-
-<?php
-// If this user has a non-empty irnpool assigned, show the pending
-// invoice reference number.
-$irnumber = getInvoiceRefNumber();
-if (!empty($irnumber)) {
-?>
- <tr>
-  <td>
-    <?php echo xlt('Tentative Invoice Ref No'); ?>:
-  </td>
-  <td>
-    <?php echo text($irnumber); ?>
-  </td>
- </tr>
-<?php
-} // Otherwise if there is an invoice reference number mask, ask for the refno.
-else if (!empty($GLOBALS['gbl_mask_invoice_number'])) {
-?>
- <tr>
-  <td>
-    <?php echo xlt('Invoice Reference Number'); ?>:
-  </td>
-  <td>
-   <input type='text' name='form_irnumber' size='10' value=''
-    onkeyup='maskkeyup(this,"<?php echo addslashes($GLOBALS['gbl_mask_invoice_number']); ?>")'
-    onblur='maskblur(this,"<?php echo addslashes($GLOBALS['gbl_mask_invoice_number']); ?>")'
-    />
-  </td>
- </tr>
-<?php
-}
-?>
-
- <tr>
-  <td colspan='2' align='center'>
-   &nbsp;<br>
-   <input type='submit' name='form_save' value='<?php echo xla('Save'); ?>' /> &nbsp;
-<?php if (empty($_GET['framed'])) { ?>
-   <input type='button' value='<?php echo xla('Cancel'); ?>' onclick='window.close()' />
-<?php } ?>
-   <input type='hidden' name='form_provider'  value='<?php echo attr($inv_provider)  ?>' />
-   <input type='hidden' name='form_payer'     value='<?php echo attr($inv_payer)     ?>' />
-   <input type='hidden' name='form_encounter' value='<?php echo attr($inv_encounter) ?>' />
-  </td>
- </tr>
-
-</table>
-</center>
+<!DOCTYPE html>
+    <head>
+            <?php Header::setupHeader(['bootstrap', 'datetime-picker','font-awesome']);?>
```

Comment:
remove bootstrap and font-awesome from above line (they are already included by default)

Reply:
looks like not done in above line; i'll do this since this should be ready for commit very soon.

Reply:
fixed

Reply:
actually @zbig01 , I'll let you do this minor fix since still issues in the interface/patient_file/front_payment.php script